include/circt/Support/Passes.h

```diff
@@ -28,13 +32,22 @@ class VerbosePassInstrumentation : public mlir::PassInstrumentation {
   int level = 0;
   const char *toolName;
 
+  void emitToolNameAndTimestamp(llvm::raw_fd_ostream &os, TimePoint &now) {
+    auto time = std::chrono::system_clock::to_time_t(now);
+    std::stringstream ss;
+    ss << std::put_time(std::localtime(&time), "%Y-%m-%d %X");
+
+    os << "[" << ss.str() << ":" << toolName << "] ";
```

Comment:
Mega-nit: how does just a space look? The ":" looked crowded to me and other logging systems, e.g., journalctl use a space here.